src/components/Integrations/Shopify/Shopify.js

```diff
@@ -138,60 +138,64 @@ const Shopify = ({ intl, dependencies: { shopifyClient } }) => {
             </>
           ) : shopifyState.isConnected ? (
             <>
-              <div className="dp-integration__block">
-                {shopifyHeader}
-                {shopifyState.shops.map((shop) => (
-                  <div key={shop.shopName} className="block dp-integration__status">
-                    <div className="status__info">
-                      <div>
-                        <StyledShopifyLogo />
-                        <div className="status__data">
-                          <ul>
-                            <li key={shop.shopName}>
-                              <p>
-                                {_('shopify.header_synchronization_date')}{' '}
-                                <strong>
-                                  <FormattedDate value={shop.synchronization_date} timeZone="UTC" />
-                                </strong>
-                              </p>
-                              <p>
-                                {_('shopify.header_store')} <strong> {shop.shopName} </strong>
-                              </p>
-                            </li>
-                          </ul>
+              {shopifyState.shops.map((shop) => (
+                <div key={shop.shopName}>
```

Comment:
On the other hand, are these keys safe? I mean, we cannot have duplicated names?

Reply:
I don't think we can, they are shopify urls to the store.

Reply:
👍